README.md

```diff
@@ -52,8 +52,8 @@ interface Promise
     /**
      * Registers a callback to be invoked when the promise is resolved.
      *
-     * The callback receives `null` as first parameter and `$value` as second parameter on success. It receives the
-     * failure reason as first parameter and `null` as second parameter on failure.
+     * The callback receives `null` as first parameter and `$value` as second parameter on success. 
+     * The callback receives reason as first parameter and `null` as second parameter on failure.
```

Comment:
`$reason` is also weird because it's nowhere used. In that case I prefer the original wording.